extensions/amp-subscriptions/0.1/test/test-platform-store.js

```diff
@@ -1,3 +1,7 @@
+import {expect} from 'chai';
```

Comment:
remove this, vscode will often auto-import it even though its not needed so you have to keep an eye on it